src/dds/rs-dds-sensor-proxy.cpp

```diff
@@ -243,7 +247,7 @@ dds_sensor_proxy::find_profile( sid_index sidx, realdds::dds_motion_stream_profi
 void dds_sensor_proxy::open( const stream_profiles & profiles )
 {
     _formats_converter.prepare_to_convert( profiles );
-
+    _target_profiles = profiles;
```

Comment:
Should implement
> void dds_sensor_proxy::close()
{
    software_sensor::close();
    _target_profiles.clear();
}

Reply:
Added